R/holiday.R

```diff
@@ -110,12 +110,14 @@ prep.step_holiday <- function(x, training, info = NULL, ...) {
   )
 }
 
-
 is_holiday <- function(hol, dt) {
-  hdate <- holiday(year = unique(year(dt)), Holiday = hol)
+  years <- unique(year(dt))
+  years <- years[!is.na(years)]
```

Comment:
You want to change 

```r
years <- years[!is.na(years)]
```

to 

```r
na_rows <- which(is.na(years))
years <- years[-na_rows]
```

It wouldn't be able to remove the `%in%` because they are different things. The year variable is only used by `holiday()` to limit the number of returned holidays.

Reply:
Yes